modules/F50_base_aggregator.sh

```diff
@@ -441,6 +446,9 @@ get_data() {
   if [[ -f "$LOG_DIR"/"$S22_LOG" ]]; then
     S22_PHP_VULNS=$(grep -a "\[\*\]\ Statistics:" "$LOG_DIR"/"$S22_LOG" | cut -d: -f2)
     S22_PHP_SCRIPTS=$(grep -a "\[\*\]\ Statistics:" "$LOG_DIR"/"$S22_LOG" | cut -d: -f3)
+    S22_PHP_INI_LIMIT_EXCEEDED=$(grep -a "\[\*\]\ Statistics:" "$LOG_DIR"/"$S22_LOG" | cut -d: -f4)
+    S22_PHP_INI_FAILURE=$(grep -a "\[\*\]\ Statistics:" "$LOG_DIR"/"$S22_LOG" | cut -d: -f5)
```

Comment:
could we count these to one php_ini_issues value? I do not think all the different types are interesting for the aggregator